### PR TITLE
two

### PR DESCRIPTION
**Pull request by Fil**
deliberate ES6 syntax

### CODE REVIEW

**Comment by enjalot** [comment]
I see some cases where `var` becomes `const` and some where `var` remains untouched.
is it intended to update to `let` or `const` in all cases?

**Comment by Fil** [comment]
> is it intended to update to `let` or `const` in all cases?

https://github.com/d3/d3/issues/3435

File: README.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ In addition to the ubiquitous and machine-friendly [RGB](#rgb) and [HSL](#hsl) c
 
 Cubehelix features monotonic lightness, while CIELAB and its polar form CIELCh<sub>ab</sub> are perceptually uniform.
```
Comment (hobbspauldevon94) [inline README.md:38]: Sounds like a cop

Comment (hobbspauldevon94) [reply README.md:38]: Just a little bit

File: package.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "d3-color",
```
Comment (hobbspauldevon94) [inline package.json:2]: Sounds like a raping studio

File: package.json [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
-  "version": "1.4.1",
+  "version": "2.0.0-rc.1",
+  "publishConfig": {
+    "tag": "next"
+  },
   "description": "Color spaces! RGB, HSL, Cubehelix, Lab and HCL (Lch).",
```
Comment (hobbspauldevon94) [inline package.json:7]: I don't have any thing to do today. It it's fine

**Pull request merged**